### PR TITLE
fix: nodelabels in tas's sample

### LINKED ISSUE #3481: [Bug] [Doc] TAS's document is wrong
**What happened**:
image

**What you expected to happen**:
I think this should be `cloud.provider.com/topology-block`

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Kueue version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### LINKED ISSUE #3482: TAS: may report negative values of pods in messages for non-admitted workloads
**What happened**:

When a workload using preferred topology annotation cannot be admitted is sometimes reports negative number of pods in the status message of the workload.

**What you expected to happen**:

negative values of pods are never reported, it does not make sense.

**How to reproduce it (as minimally and precisely as possible)**:

I have a reproducible scenario, but it is yet not simplified enough to be included here. I will simplify and post.

Basically I create multiple jobs using preferred, and workload corresponding to the non-admitted job reports message like:
```
status:
  conditions:
  - lastTransitionTime: "2024-11-07T13:01:01Z"
    message: 'couldn''t assign flavors to pod set main: insufficient unused quota
      for cpu in flavor tas-flavor, 30 more needed, topology "gke-default" allows
      to fit only -18 out of 40 pod(s)'
    observedGeneration: 1
    reason: Pending
    status: "False"
    type: QuotaReserved
```

**Anything else we need to know?**:

It was reported first here: https://github.com/kubernetes-sigs/kueue/issues/3211#issuecomment-2458900114

### PR DESCRIPTION
**Pull request by KunWuLuan**
#### Which issue(s) this PR fixes:

Fixes #3481

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/static/examples/tas/sample-queues.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ metadata:
   name: "tas-flavor"
 spec:
   nodeLabels:
-    cloud.provider.com/: "tas-node-group"
+    cloud.provider.com/topology-block: "tas-node-group"
```
Comment (mimowo) [inline site/static/examples/tas/sample-queues.yaml:17]:
```suggestion
    cloud.provider.com/node-group: "tas-node-group"
```
To better capture the intention as described in https://github.com/kubernetes-sigs/kueue/issues/3481#issuecomment-2464248854. 

cc @mwysokin

**Comment by mbobrovskyi** [comment]
/release-note-edit 

```release-note 
NONE
```

**Comment by mbobrovskyi** [comment]
/lgtm
Thanks!

**Comment by mimowo** [comment]
/approve
/cherry-pick website
Thanks!

**Comment by mbobrovskyi** [comment]
Do we need to cherry-pick?

**Pull request merged**